cla_backend/apps/legalaid/models.py

```diff
@@ -687,7 +718,7 @@ def split(self, user, category, matter_type1, matter_type2, assignment_internal)
                     'callback_attempt', 'search_field', 'provider_assigned_at'
                 ],
                 'clone_fks': [
-                    'thirdparty_details', 'adaptation_details'
+                    'thirdparty_details', 'adaptation_details', 'eod_details',
```

Comment:
probably shouldn't be cloned when case is copied so can add 'eod_details' to 'excludes' = {... above and remove it from 'clone_fks' = {...